org.eclipse.scout.rt.ui.html/src/main/java/org/eclipse/scout/rt/ui/html/json/form/fields/JsonAdapterProperty.java

```diff
@@ -44,8 +44,11 @@ protected IUiSession getUiSession() {
     return m_uiSession;
   }
 
-  protected boolean isGlobal() {
-    return m_global;
+  /**
+   * @return true whether a global adapter should be created for the given model.
+   */
+  protected boolean isGlobal(Object model) {
```

Comment:
Do we need to keep the no-arg-method? Projects might implement own `JsonAdapterProperty`s or override the method instead of using `createConfig()`.